docs/reference/eql/syntax.asciidoc

```diff
@@ -847,81 +847,190 @@ The query's event items correspond to the following states:
 * State B:  `[process where process.name == "bash"]`
 * Complete: `[process where process.name == "cat"]`
 
+[source,txt]
+----
++------------------------------------------------------+
+|  +-----------+     +-----------+     +------------+  |
+|  |  State A  |     |  State B  |     |  Complete  |  |
+|  +-----------+ --> +-----------+ --> +------------+  |
+|  |           |     |           |     |            |  |
+|  +-----------+     +-----------+     +------------+  |
++------------------------------------------------------+
+----
+
 To find matching sequences, the query uses separate state machines for each
-unique `user.name` value. Pending sequence matches move through each machine's
-states as follows:
+unique `user.name` value. Based on the data set, you can expect two state
+machines: one for the `root` user and one for `elkbee`.
 
 [source,txt]
 ----
-{ "index" : { "_id" : "1" } }
++------------------------"root"------------------------+
+|  +-----------+     +-----------+     +------------+  |
+|  |  State A  |     |  State B  |     |  Complete  |  |
+|  +-----------+ --> +-----------+ --> +------------+  |
+|  |           |     |           |     |            |  |
+|  +-----------+     +-----------+     +------------+  |
++------------------------------------------------------+
+
++-----------------------"elkbee"-----------------------+
+|  +-----------+     +-----------+     +------------+  |
+|  |  State A  |     |  State B  |     |  Complete  |  |
+|  +-----------+ --> +-----------+ --> +------------+  |
+|  |           |     |           |     |            |  |
+|  +-----------+     +-----------+     +------------+  |
++------------------------------------------------------+
+----
```

Comment:
I toyed around with creating fancier SVG images for this section. For example

![separate-state-machines](https://user-images.githubusercontent.com/40268737/101195442-6b815b80-362d-11eb-8b93-ec31a1206e3c.jpg)

However, they broke up the later event-by-event example snippet, and I felt that presenting the diagram in two formats could be confusing. ASCII diagrams (thanks for the idea, Costin!) felt like a better fit. Let me know if you feel differently!